src/cli.js

```diff
@@ -39,6 +39,12 @@ program
   .option('--no-hmr', 'disable hot module replacement')
   .option('--no-cache', 'disable the filesystem cache')
   .option('--no-source-maps', 'disable sourcemaps')
+  .option('--no-autoinstall', 'disable autoinstall')
+  .option(
+    '-m, --package-manager <packageManager>',
```

Comment:
Is this option really necessary? Our approach to inferring the package manager seems pretty solid.

Reply:
That’s true, but I don’t really see any downsides to giving the users manual control if necessary.

One scenario that I would imagine will be particularly problematic is if a user is starting a brand new project with Parcel and has no dependencies yet. They try to add their first import statement, and Parcel automatically uses npm because it doesn’t see a `yarn.lock`, even though the user would rather use yarn. In that case they would want to use the `-m yarn` flag to force it to use yarn. 

(I mean I suppose they could just `touch yarn.lock` to generate an empty one, but that’s kinda hacky and just another point for confusion)

I can remove it if you think we should, but I just don’t see any downsides to leaving it.

Reply:
Going to remove it for now. I think our approach is pretty solid. If someone wants to open a bug with a good reason to add an option for it, they can.